Dockerfile-ci

```diff
@@ -0,0 +1,23 @@
+#  Copyright 2015 Daniel Giribet <dani - calidos.cat>
```

Comment:
Can we use same Dockerfile as we are using on Jenkins? Take a look at [support/docker_build.sh](https://github.com/apache/mesos/blob/42e8d6fb8b894c08b4b0c7cc40036fe630489bba/support/docker_build.sh)